sklearn/tests/test_pipeline.py

```diff
@@ -1035,6 +1072,372 @@ def test_pipeline_memory():
         shutil.rmtree(cachedir)
 
 
+def test_pipeline_memory_resampler():
+    X, y = make_classification(
+        n_classes=2,
+        class_sep=2,
+        weights=[0.1, 0.9],
+        n_informative=3,
+        n_redundant=1,
+        flip_y=0,
+        n_features=20,
+        n_clusters_per_class=1,
+        n_samples=500,
+        random_state=0)
+    cachedir = mkdtemp()
+    try:
+        memory = Memory(cachedir, verbose=10)
+        # Test with Transformer + SVC
+        clf = SVC(gamma='scale', probability=True, random_state=0)
+        transf = DummyResampler()
```

Comment:
You can pack a bit more the parameters.

Reply:
Sorry, I don't understand what you mean.